turbinia/workers/photorec.py

```diff
@@ -44,21 +43,21 @@ def run(self, evidence, result):
     output_evidence = PhotorecOutput()
     # Create a path that we can write the new file to.
     base_name = os.path.basename(evidence.local_path)
-    output_file_path = os.path.join(self.output_dir, base_name)
+    output_file_path = os.path.join(self.output_dir, 'photorec_finding')
```

Comment:
Very minor nit, but maybe 'photorec_ouput' (or if you wanted to keep finding, we could make it 'photorec_findings')?

Reply:
Done.